frontend_app_collaborative_document_edition/test/CollaboraEditionFrame.spec.js

```diff
@@ -20,8 +20,25 @@ describe('<CollaborativeEditionFrame />', function () {
       config: {
         apiUrl: 'http://unit.test:6543/api/v2',
         translation: { en: { translation: {} } },
+        system: {
+          appListLoaded: true,
+          config: {
+            content_length_file_size_limit: 0,
+            email_notification_activated: true,
+            instance_name: "Algoo",
```

Comment:
Why not choose `Tracim` in place of `Algoo` ?

Reply:
mmh, it's unit test data, I didn't thought a lot about it